test/odbc/psqlodbc/test/image.cpp

```diff
@@ -0,0 +1,145 @@
+#include "../conversion_functions_common.h"
+#include "../psqlodbc_tests_common.h"
+#include <fstream>
+
+const string TABLE_NAME = "master_dbo.image_table_odbc_test";
+const string COL1_NAME = "pk";
+const string COL2_NAME = "data";
+const string DATATYPE_NAME = "sys.image";
+const string VIEW_NAME = "master_dbo.image_view_odbc_test";
+
+// Relative to where `./build/main` is ran
+const std::string INSERTED_FILE_PATH = "psqlodbc/Images/Black.png";
+const std::string UPDATED_FILE_PATH = "psqlodbc/Images/circle.png";
+static string INSERTED_IMAGE_HEX_STR;
+static string UPDATED_IMAGE_HEX_STR;
```

Comment:
Can these be const?

Reply:
+= won't be able to handle const but static